value-fixture/pom.xml

```diff
@@ -172,27 +172,83 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>${maven-compiler-plugin.version}</version>
         <configuration>
-          <compilerId>javac-with-errorprone</compilerId>
-          <forceJavacCompilerUse>true</forceJavacCompilerUse>
           <compilerArgs>
-            <arg>-Xep:CheckReturnValue:ERROR</arg>
-            <arg>-Xep:MethodCanBeStatic:ERROR</arg>
+            <arg>-XDcompilePolicy=simple</arg>
+            <arg>-Xplugin:ErrorProne -Xep:CheckReturnValue:ERROR -Xep:MethodCanBeStatic:ERROR -Xep:UnnecessaryCheckNotNull:WARN</arg>
             <arg>-Xlint:unchecked</arg>
           </compilerArgs>
+          <annotationProcessorPaths>
```

Comment:
that is probably why import rewrite and custom annotation extension do not get into classpath of the annotation processing. The question can we switch back to a regular classpath here? Or alternatively we probably need to package test extensions as a separate artifact and then include it here in a annotationProcessorPaths in order for mentioned tests to work.